zebra-state/src/service/finalized_state.rs

```diff
@@ -205,7 +206,13 @@ impl FinalizedState {
                 return batch;
             }
 
-            // Index each transaction
+            // Index all new transparent outputs
+            for (outpoint, utxo) in new_outputs.into_iter() {
+                batch.zs_insert(utxo_by_outpoint, outpoint, utxo);
+            }
+
+            // Index each transaction, spent inputs, nullifiers
+            // TODO: move computation into FinalizedBlock as with transparent outputs
```

Comment:
Nope, there's no issue for it.  I put the TODO in the source rather than creating a separate tracking issue as an explanation and reminder, so that ideally when future work edits this part of the code, the author feels empowered to move in that direction as part of their future work.  I'd be happy to create a separate issue for it if you'd like, but my thought was that it wasn't an important enough task to devote attention to, beyond a reminder for opportunistic edits.